test_unstructured/partition/test_msg.py

```diff
@@ -379,6 +379,24 @@ def it_adds_email_specific_fields_to_the_msg_element_metadata(self, opts_args: d
         assert opts.msg_metadata.sent_to == ["mrobinson@unstructured.io"]
         assert opts.msg_metadata.subject == "Test Email"
 
+    def it_captures_cc_and_bcc_element_metadata(self, opts_args: dict[str, Any]):
+        opts_args["file_path"] = example_doc_path("fake-email-with-cc-and-bcc.msg")
+        opts = MsgPartitionerOptions(**opts_args)
+
+        assert opts.msg_metadata.cc_recipient == ["steve@unstructured.io"]
+        assert opts.msg_metadata.bcc_recipient == ["hello@unstructured.io"]
+        assert opts.msg_metadata.sent_to == [
+            "john-ctr@unstructured.io",
+            "steve@unstructured.io",
+            "hello@unstructured.io",
+        ]
+
+    def it_captures_message_id_element_metadata(self, opts_args: dict[str, Any]):
+        opts_args["file_path"] = example_doc_path("fake-email-with-cc-and-bcc.msg")
+        opts = MsgPartitionerOptions(**opts_args)
+
+        assert opts.msg_metadata.message_id == "<14DDEF33-2BA7-4CDD-A4D8-E7C5873B37F2@gmail.com>"
```

Comment:
Should the angle brackets stay or be removed for `message_id`?

Reply:
Based on this from the doc, I'd say remove the angle brackets

> Message-ID consists of a long string of characters (unique identifier of this message on the mail server) ending with the FQDN of the mail server (e.g. 25eb7e48-2252-48ea-80ce-cf2f6119a8e3@ENV95-E2013-1.domain95.lab).

Reply:
Done :)